modules/client-configuration/pages/clients-amazon.adoc

```diff
@@ -19,7 +19,7 @@ Traditional clients are not available on {amazon}{nbsp}2.
 
 [WARNING]
 ====
-{amazon} instances always have the same `machine-id` id at `/etc/machine-id`.
```

Comment:
You are missing this WARNING for Almalinux, aren't you?

Reply:
added.